### ISSUE TITLE
Git - Conflict notification should be actionable 

### ISSUE THREAD

**Issue by daviddossett** [microsoft/vscode#153712]
Feels like this should have a button to open the merge editor. Thoughts @jrieken @hediet @isidorn?

<img width="529" alt="CleanShot 2022-06-29 at 09 00 30@2x" src="https://user-images.githubusercontent.com/25163139/176482553-79771f00-20fe-4195-a73f-3fbe86914634.png">

**Comment by jrieken** [comment]
Yeah, that's a blast from the past when there was no merge editor. It is a good opportunity to "sell" that now

**Comment by isidorn** [comment]
@daviddossett aha, so if it can just happen with you merging branches from terminal than a dialog is too much. I agree that the notification is better.

**Comment by jrieken** [comment]
Low-hanging: add command to "Open SCM View"

**Comment by jrieken** [comment]
@lszomoru leaving this up to SCM land. This is specific to 3wm and I don't even know how to trigger this...

**Comment by lszomoru** [comment]
Updated the notification to be more actionable:
<img width="464" alt="Image" src="https://github.com/user-attachments/assets/0ad9f487-f51c-45e6-b400-553090ff8c00" />

**Comment by osortega** [comment]
Not sure if I'm doing the steps right since I'm getting a modal kind of notification, but I don't see the `Show Changes` button.

The steps I did were:
1. Change a file in the main branch directly through github.com
2. Commit a conflicting change through the local cloned repo
3. Try to push the change to the remote through source control

<img width="1200" alt="Image" src="https://github.com/user-attachments/assets/66054bc3-b05e-477f-85a8-e361a1200a93" />
